app/views/services/govbox/register_step2.html.erb

```diff
@@ -7,27 +7,62 @@
         <h1 class="form-title">Údaje štatutára</h1>
         <p class="form-subtitle lead">Zadajte údaje štatutára (napríklad konateľa s.r.o.), ktorý podpíše splnomocnenie na prístup k schránke.</p>
 
-        <div class="row">
-          <div class="col-md-4 form-group">
-            <%= label_tag :given_name, 'Meno' %>
-            <%= text_field_tag :given_name, params[:given_name], class: 'form-control input-lg', required: true %>
+        <% if @statutory_entries %>
+          <h6 style="margin: 30px 0;">Vyberte štatutára zo zoznamu:</h6>
+          <div class="row">
+            <div class="col-md-12">
+              <% @statutory_entries.each_with_index do |entry, index| %>
+                <label class="form-check-label" style="margin: 8px 0;">
+                  <%= radio_button_tag :statutory_entry, index, false, class: 'form-check-input', data: { given_name: entry[:first_name], family_name: entry[:last_name], person_formatted_address: entry[:formatted_address] } %>
+                  <span><%= entry[:label] %></span>
+                </label>
+              <% end %>
+            </div>
           </div>
-          <div class="col-md-5 form-group">
-            <%= label_tag :family_name, 'Priezvisko' %>
-            <%= text_field_tag :family_name, params[:family_name], class: 'form-control input-lg', required: true %>
+          <h6 style="margin: 30px 0;">
+            <a href="#" data-toggle="collapse" data-target="#form-input-fields">alebo kliknite sem a zadajte údaje ručne</a>
+          </h6>
+
+          <script>
+            $('input:radio').change(e => {
+              let selected = e.target.dataset;
+              $('#given_name').val(selected.givenName);
+              $('#family_name').val(selected.familyName);
+              $('#person_formatted_address').val(selected.personFormattedAddress);
+            });
+
+            $(document).ready(function () {
+              $("#form-input-fields").addClass('collapse hide');
```

Comment:
poznamka pre mna: tuto to moze poskocit, treba pozriet lokalne

Reply:
@michal-rohacek ak su zle umiestnene margin/padding tak to niekedy spravi collapse a potom to este poskoci (tak naraz zmizne nejaka medzera okolo collapsnuteho elementu), deje sa to?

Reply:
nie, nevidim tam nic take